File: unused/mock-files/mock_slack_webhook.py
#!/usr/bin/env python3
from flask import Flask, request, jsonify
import json
from datetime import datetime

app = Flask(__name__)

@app.route('/webhook/slack', methods=['POST'])
def slack_webhook():
    data = request.get_json() or {}
    
    print("\n" + "="*60)
    print("🚨 CONTAINER ALERT - SLACK NOTIFICATION")
    print("="*60)
    print(f"📅 Time: {datetime.now().isoformat()}")
    print(f"📦 Container: {data.get('container', 'Unknown')}")
    print(f"⚠️  Status: {data.get('status', 'Unknown')}")
    print(f"🔍 Issue: {data.get('issue', 'Unknown')}")
    print(f"📝 Message: {data.get('text', data.get('message', 'No message'))}")
    print("="*60)
    print("✅ Alert would be sent to #alerts channel")
    print("="*60 + "\n")
    
    return jsonify({
        "ok": True,
        "message": "Alert sent to #alerts channel",
        "timestamp": datetime.now().isoformat(),
        "channel": "#alerts"
    })

@app.route('/health', methods=['GET'])
def health():
    return jsonify({"status": "Mock Slack webhook is running!"})

if __name__ == '__main__':
    print("🔔 Starting Mock Slack Webhook Server...")
    print("📡 Webhook URL: http://localhost:3001/webhook/slack")
    print("🎯 Ready to receive container alerts!")
    app.run(host='0.0.0.0', port=3001, debug=False)

File: unused/mock-files/mock-slack-webhook.js
const express = require('express');
const app = express();
const port = 3001;

app.use(express.json());

// Mock Slack webhook endpoint
app.post('/webhook/slack', (req, res) => {
  console.log('\n🚨 SLACK ALERT RECEIVED:');
  console.log('='.repeat(50));
  console.log('📅 Time:', new Date().toISOString());
  console.log('📦 Container:', req.body.container || 'Unknown');
  console.log('⚠️  Status:', req.body.status || 'Unknown');
  console.log('🔍 Issue:', req.body.issue || 'Unknown');
  console.log('📝 Message:', req.body.text || req.body.message || 'No message');
  console.log('='.repeat(50));
  
  // Simulate Slack response
  res.json({
    ok: true,
    message: 'Alert sent to #alerts channel',
    timestamp: new Date().toISOString()
  });
});

// Health check
app.get('/health', (req, res) => {
  res.json({ status: 'Mock Slack webhook is running!' });
});

app.listen(port, () => {
  console.log(`🔔 Mock Slack webhook running at http://localhost:${port}`);
  console.log(`📡 Webhook URL: http://localhost:${port}/webhook/slack`);
  console.log('Ready to receive container alerts!');
});